content/zh/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-deprecation-affects-you.md

```diff
@@ -0,0 +1,158 @@
+   for more information.
+4. Make sure there is no indirect dependencies on dockershim behavior.
+   This is an edge case and unlikely to affect your application. Some tooling may be configured
+   to react to Docker-specific behaviors, for example, raise alert on specific metrics or search for
+   a specific log message as part of troubleshooting instructions.
+   If you have such tooling configured, test the behavior on test
+   cluster before migration.
+ -->
+1. 确认没有特权 pod 执行 docker 命令。
+2. 检查 Kubernetes 基础架构外部节点上的脚本和应用，确认它们没有执行 Docker 命令。可能的命令有：
+   - SSH 到节点排查故障；
+   - 节点启动脚本；
+   - 直接安装在节点上的监视和安全代理。
+3. 执行了上面提及的特权操作的第三方工具。详细操作请参考:
+   [从 dockershim 迁移遥测和安全代理](/zh/docs/tasks/administer-cluster/migrating-from-dockershim/migrating-telemetry-and-security-agents)
+4. 确认没有对 dockershim 行为的间接依赖。这是一种极端情况，不太可能影响你的应用。
+   一些工具很可能被配置为使用了 Docker 特性，比如，基于特定指标发警报，或者在排障指令的一个环节中搜索特定的日志信息。
+   如果你有此类配置的工具，需要在迁移之前，在测试集群上完成功能验证。
+
+
+<!-- ## Dependency on Docker explained {#role-of-dockershim}  -->
+## Docker 依赖的分析  {#role-of-dockershim}
+
+<!-- 
+A [container runtime](/docs/concepts/containers/#container-runtimes) is software that can
+execute the containers that make up a Kubernetes pod. Kubernetes is responsible for orchestration
+and scheduling of Pods; on each node, the {{< glossary_tooltip text="kubelet" term_id="kubelet" >}}
+uses the container runtime interface as an abstraction so that you can use any compatible
+container runtime.
+ -->
+[容器运行时](/zh/docs/concepts/containers/#container-runtimes)是一个软件，它用来运行组成 Kubernetes pod 的容器。
+Kubernetes 负责编排和调度 Pod；在每一个节点上，
+{{< glossary_tooltip text="kubelet" term_id="kubelet" >}}
+使用抽象的容器运行时接口，所以你可以选用任意兼容的容器运行时。
+
+<!-- 
+In its earliest releases, Kubernetes offered compatibility with just one container runtime: Docker.
+Later in the Kubernetes project's history, cluster operators wanted to adopt additional container runtimes.
+The CRI was designed to allow this kind of flexibility - and the kubelet began supporting CRI. However,
+because Docker existed before the CRI specification was invented, the Kubernetes project created an
+adapter component, `dockershim`. The dockershim adapter allows the kubelet to interact with Docker as
+if Docker were a CRI compatible runtime.
+ -->
+在初始版本中，Kubernetes 提供的兼容性只支持一个容器运行时：Docker。
+在 Kubernetes 发展历史中，集群运营人员希望适配更多的容器运行时。
+于是 CRI 被设计出来满足这类灵活性 - 而 kubelet 开始支持 CRI。
+然而，因为 Docker 在 CRI 规范创建之前就已经存在，Kubernetes 就创建了一个适配器组件：`dockershim`。
+dockershim 适配器允许 kubelet 与 Docker交互，就好像 Docker 是一个 CRI 兼容的运行时一样。
+
+<!-- 
+You can read about it in [Kubernetes Containerd integration goes GA](/blog/2018/05/24/kubernetes-containerd-integration-goes-ga/) blog post.
+ -->
+你可以阅读博文
+[Kubernetes 容器集成功能的正式发布](/zh/blog/2018/05/24/kubernetes-containerd-integration-goes-ga/)
+
+![Dockershim vs. CRI with Containerd](/images/blog/2018-05-24-kubernetes-containerd-integration-goes-ga/cri-containerd.png)
```

Comment:
nit: It's OK to localize the alt attribute here.

Reply:
cool, It's upexpected that you review the Chinese translation!
I would like to ask a question, do I need to add / zh in front of the address of this picture, and then copy the pic to Chinese repo.